java/code/src/com/redhat/rhn/frontend/xmlrpc/LoggingInvocationProcessor.java

```diff
@@ -43,6 +45,43 @@ protected synchronized Object initialValue() {
         }
     };
```

Comment:
Timer implementation can be moved to the base class while exposing necessary methods to start/stop from the subclasses.